src/App.jsx

```diff
@@ -15,6 +16,9 @@ function App() {
     <Route path="/articles" element={<Articles />} />
     <Route path='/articles/:id' element={<Article />} />
     <Route path='/login' element={<Login />} />
+    <Route path="*" element={<Error
```

Comment:
this is a great start! However, since you haven't set up any kind of homepage or landing page on the "/" endpoint, when the user first navigates to your site they see an error page which isn't a good user experience and might cause users to navigate away